iam-migrate.go

```diff
@@ -26,9 +26,11 @@ import (
 	"net/http"
 )
 
-type SkippedIAMEntities struct {
-	SkippedAccessKeys []string `json:"skippedAccessKeys,omitempty"`
-	SkippedDN         []string `json:"skippedDN,omitempty"`
+type ImportIAMResult struct {
+	Skipped []string `json:"skipped,omitempty"`
```

Comment:
Is it going to be string alone? Isn't IAM various types of assets?

Reply:
accesskey and username are strings so kept like this